src/commons/RestAPI.js

```diff
@@ -5,6 +5,36 @@ import axios from 'axios';
 import { geometryDAGMA } from '../assets/mockups/summaryData';
 
 class RestAPI {
+
+  /** **************** */
+  /** HOME INFORMATION */
+  /** **************** */
+
+  /**
+   * Request anh areas list
+   *
```

Comment:
Please delete alll these extra lines at the end of documentation (lines 15, 24, 32)